tsconfig.base.json

```diff
@@ -1207,6 +1207,13 @@
     "strict": true,
     // for now, don't use unknown in catch
     "useUnknownInCatchVariables": false,
+    // enabled to point out not supported features when emitting in an isolated module environment
+    "isolatedModules": true,
+    // TODO(jbudz): should be uncommented when upgrading to TS@4.8
+    //
+    // ensures every non-declaration file is treated as a module
+    // "moduleDetection": "force",
+    //
```

Comment:
We don't need to comment all this out, right? It doesn't do anything yet but it will as soon as we upgrade.

Reply:
I commented it out because not doing it was failing some CI step (cant remember which one)